### ISSUE TITLE
Compile just one fs/fsx file

### ISSUE THREAD

**Issue by dredgy** [thinkbeforecoding/peeble#1]
Hi, first off - love the project. I think F# is great syntax for scripting and being able to use it with ease of use of PHP is awesome. Can't wait to see where this goes.

Tiny feature request, if possible to just compile one file at a time instead of the whole project. Is just a bit quicker in workflow as I have the compiler set to run every time I save a file.

Cheers

**Comment by thinkbeforecoding** [comment]
Yes that would be definitely doable to pass only on fsx file, especially if its standalone.

**Comment by dredgy** [comment]
Hmm, I use the feature a lot in fable. But the more I build out peeble the less I need it. So far loving it.

Now to just get the entire PHP standard library to work by using “open PHP” 😋

**Comment by thinkbeforecoding** [comment]
Super happy that it's useful for you.
I've not ported all FSharp.Core yet, this is a lot of work, but don't hesitate to contribute.
I was thinking also about integrating it with Composer for instance like Fable do it with npm.

My primary use was to port my boardgame CrazyFarmers I wrote in F# to Php to port it to Board Game Arena  and avoid rewriting everything, given the front was using Fable with the same code.

What kind of project do you use Peeble on ?

**Comment by dredgy** [comment]
I’m using it do develop a point of sale system for fine dining restaurant  which was already partially in PHP (and a lot of JS) so I’m the opposite of you, I’m trying to come from PHP but prefer F#s syntax. My JavaScript is a little too specialised to port over to Fable without a bunch of bindings work, but the PHP is minimal, I just do t want to switch from an easily deplorable Apache server and so far all seems good. I’m a beginner with F# though so who knows where it goes.
